config/locales/de.events.yml

```diff
@@ -10,6 +10,7 @@ de:
       filter: "Filtern"
       status: "Status"
       details: "Details"
+      age_when_event_starts: "Alter <br> bei Beginn"
```

Comment:
Yeah..I wasn't sure either^^
But I'm also not very sure if the fix is that much better..